src/promptflow-tracing/pyproject.toml

```diff
@@ -36,17 +36,20 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<4.0,>=3.8"
+Jinja2 = "*"
 openai = "*"
```

Comment:
openai>1.0

Reply:
this dependency comes from openai injector, and we support all version, so no need to add boundary.